app/views/dvr/dvrFiles.scala.html

```diff
@@ -74,56 +70,62 @@
     }
 }
 
-@valuationTabContent = @{valuationTab(model, evaluateRoute)(govukButton, govukInsetText, govukWarningText)}
-
-@startCheckTabContent = @{
-    startCheckTab(model, startCheckForm)(govukButton, govukDetails, govukRadios, formWithCSRF)
-}
-
-@checkCasesTabContent = @{
-    checkCasesDetailsTab(model.submissionId, model.owner, model.uarn, model.authorisationId, model.valuationId, model.listYear, checkCases, evaluateRoute(model.check), "#start-check-tab")(govukButton, govukTable, govukDetails, govukWarningText)
-}
-
-@challengeCasesTabContent = @{
-    challengeCasesDetailsTab(model.authorisationId, challengeCases, model.owner, model.submissionId, model.valuationId, model.uarn)(govukTable, govukDetails)
-}
-
-@agentsTabContent = @{agentsTab(model)(govukTable)}
-
 @tabs = @{
    val valuation: TabItem = TabItem(
         id = Some("valuation-tab"),
         label = messages("available.requestvaluation.valuationTab.title"),
         panel = TabPanel(
-            content = HtmlContent(valuationTabContent)
+            content = HtmlContent(valuationTab(model, evaluateRoute))
         )
     )
     val startCheck: TabItem = TabItem(
         id = Some("start-check-tab"),
         label = messages("available.requestvaluation.startCheckTab.title"),
         panel = TabPanel(
-            content = HtmlContent(startCheckTabContent)
+            content = HtmlContent(startCheckTab(model, startCheckForm))
         )
     )
     val check: TabItem = TabItem(
         id = Some("check-cases-tab"),
         label = checkCasesTabTitle,
         panel = TabPanel(
-            content = HtmlContent(checkCasesTabContent)
+            content = HtmlContent(
+                checkCasesDetailsTab(
+                    model.submissionId,
+                    model.owner,
+                    model.uarn,
+                    model.authorisationId,
+                    model.valuationId,
+                    model.listYear,
+                    checkCases,
+                    evaluateRoute(model.check),
+                    "#start-check-tab",
+                    checkSummaryUrl
+                )
+            )
         )
     )
     val challenge: TabItem = TabItem(
             id = Some("challenge-cases-tab"),
             label = messages("available.requestvaluation.challengesTab.title", challengeCases.size),
             panel = TabPanel(
-              content = HtmlContent(challengeCasesTabContent)
+              content = HtmlContent(
+                  challengeCasesDetailsTab(
+                      model.authorisationId,
```

Comment:
same sort of thing here, picking all sorts of things out of the model, when you could just pass the model